### DISCUSSION TITLE
Custom local exercises

### DISCUSSION THREAD

**Discussion by ddakotac** [wger-project/wger#1575] [Q&A]
Hello! I just installed Wger and am enjoying exploring it. I have installed a local docker image.
I see that there is an option to "contribute an exercise", but that is locked for an account under 21 days old, and what I was hoping to do is create a custom, local-only exercise. Is this possible?
EG I would like to create an exercise simply called "stretch" so I can see what days I did that and for how long.
I'm fairly certain I've exhausted every corner of the app, and the documentation doesn't seem to capture anything on the end-user side.
Thanks for any help!

Edit - I see that there is an option in the .env file to change the age of an account before it can add a custom exercise - when I skimmed over that before I thought it was something else - so I think editing that will fit my needs. For some reason the wording of "contribute an exercise" made me think it would be uploading it to a community database or something. But I now surmise that this is actually contributing to my "local" exercise collection since the age is a variable I can set? 
Though I have tried setting MIN_ACCOUNT_AGE_TO_TRUST=0 as well as commenting it out, but restarting the containers does not seem to update this in the app. It is It's also saying I have to verify my email, can this be disabled?

Thanks!

**Comment by ddakotac** [comment]
Also on another note, at least on the website version I don't actually see a way to log an actual workout from the main page - I have to go to the calendar - most of the UI seems to be around setting up the routines. It seemed a bit roundabout to do it on the mobile aspect as well. Is there something I'm missing here on the workout logging aspect of Wger?

Regards

**Comment by rolandgeider** [comment]
Hi! If you have a local instance, the admin should be able to add any exercise to the local database immediately. I see how the wording might be a bit confusing if you are the only one using it, but nothing from your server leaves it (the only communication that happens by default with the docker setup is that exercises from the "official" server get synchronized to yours)

To log a workout (without using the gym mode in the mobile app) you need to open the detail page of the workout and from there you can save logs from the individual days (click on the three dots)

**Reply by ddakotac** [reply]
Thank you for the guidance and for the project! I must not be using the admin account then. I will continue to experiment.
Edit - I can confirm that logging in with the default admin account allowed me full control over adding exercises. Thanks again.
